### ISSUE TITLE
[Bug] Multiple seeding credit for single torrent

### ISSUE THREAD

**Issue by ChristianGroeber** [HDInnovations/UNIT3D-Community-Edition#1390]
**Describe the bug**
I accidentally had the same torrent on 2 drives and registered in 2 clients and now I get double seeding credit for said torrent. Even though it was uploaded to the site 3 months ago, I get seeding credit for 6 months.

**To Reproduce**
1. Download a .torrent file
2. Start downloading
3. Once completed, copy the downloaded files to another location
4. Open a second torrent client, add the .torrent file and tell the client where to look for the files
5. See your Seeding credit skyrocket

**Expected behavior**
The Tracker should realize it's the same user, seeding from multiple clients

**UNIT3D (please complete the following information):**
- UNIT3D Version: unknown
- Laravel Version: unknown
- PHP Version: unknown
- Database Driver & Version: unknown
- Web Server Driver & Version: unknown
- OS Driver and Version: unknown

**Desktop (please complete the following information):**
 - OS: Linux Ubuntu
 - Browser: Google Chrome
 - Version: 80.0.3987.149

**Additional context**
-

**Possible Fix?**
-

**Comment by HDVinnie** [comment]
You mean in relation to BON accumulation?

**Comment by ChristianGroeber** [comment]
No, it doesn't give twice the BON for one torrent, but credits double the seedtime. So as a result, yeah, it means more BON because the site thinks I've been seeding for 6 months which gives more BON, whereas, in reality, I've only been seeding for 3 months.

**Comment by PyR8zdl** [comment]
Hmm. I wonder if we should be using History->updated_at instead of Peer since there can be multiple Peer instances of the same user.

https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/9aefd14481594b86d1c40e956a16de1103a4e7f5/app/Http/Controllers/AnnounceController.php#L283
